app/src/main/java/com/vietdung/mvparchitect/SearchUserContract.java

```diff
@@ -0,0 +1,11 @@
+package com.vietdung.mvparchitect;
+
+public interface SearchUserContract {
+    interface View{
+        void clearForcus();
+    }
+
+    interface Presenter{
+        void searchUser();
```

Comment:
Truyền param query vào hàm search
```suggestion
        void searchUser(String query);
```